### PR TITLE
Use awesome-phonenumber to format phone numbers

### PR DESCRIPTION
**Pull request by RockyTV**
I haven't had much experience with Vue but I believe it should work?

Anyways, the reasoning behind this PR is to format phone numbers according to their country code. This library, [awesome-phonenumber](https://github.com/grantila/awesome-phonenumber) is a user-friendly version of Google's `libphonenumber`, used in Android and ported to JavaScript.

**EDIT**: I chose `awesome-phonenumber` because its small and faster than the alternatives based on Google's library.

### CODE REVIEW

File: components/mixins/place.js [hunk 1]
```diff
@@ -14,11 +14,9 @@ export default {
     },
 
     contact() {
+      var PhoneNumber = require('awesome-phonenumber');
```
Comment (francois2metz) [inline components/mixins/place.js:17]: You should move it to the top of the `<script>` tag and use the modern import syntax.

**Comment by francois2metz** [comment]
That's a good idea!

File: components/mixins/place.js (hunk 1 above)
Comment (RockyTV) [reply components/mixins/place.js:17]: You mean moving the import to the top of the file, or moving it to another file?

**Comment by francois2metz** [comment]
Merged in 72f738c22ee379f1980fb26104bfac5531dd7a12. Thanks.

**Pull request closed without merging**